public/components/ContentPanel/ContentPanel.tsx

```diff
@@ -22,11 +22,16 @@ import {
   EuiHorizontalRule,
   EuiPanel,
   EuiTitle,
+  EuiText,
+  EuiBadge,
 } from '@elastic/eui';
 
 type ContentPanelProps = {
   title: string;
   titleSize?: EuiTitleSize;
+  isLive?: boolean;
```

Comment:
Same feeling, here is to add some icon after the panel title, similarly we may add some link or others. How about we make it more general like titleSuffix?

Reply:
Or even better, how about we accept node to `displayTitl e`  ? Consumer can define their own way of displaying title. So it could be either String (Current behavior) or A node consumer takes over control.

Reply:
I'm ok for that, actually I used your solution for subtitle to just display anything passed to ContentPanel (in my case, I need to add some links), so we can get most flexibility. For our current use cases, we just add live icon or some link to title, will result in some duplicate code, but I'm ok about that.

Reply:
> Or even better, how about we accept node to `displayTitle` ? Consumer can define their own way of displaying title. So it could be either String (Current behavior) or A node consumer takes over control.

I think we can accept `title` as either string or a node, to make it backwards compatible.

@mihirsoni and @ylwu-amzn feel free to let me know if that sounds good to you

Reply:
I have a similar problem as well with the title here: https://aws-uxdr.invisionapp.com/share/MZVZS92PYSX#/screens/405140830. I have a soln right now which is similar to Yizhe's by adding a field in the ContentPanelProps. Let me know what you guys decide on.

Reply:
> > Or even better, how about we accept node to `displayTitle` ? Consumer can define their own way of displaying title. So it could be either String (Current behavior) or A node consumer takes over control.
> 
> I think we can accept `title` as either string or a node, to make it backwards compatible.
> 
> @mihirsoni and @ylwu-amzn feel free to let me know if that sounds good to you

Sounds good for me.

Reply:
> > Or even better, how about we accept node to `displayTitle` ? Consumer can define their own way of displaying title. So it could be either String (Current behavior) or A node consumer takes over control.
> 
> I think we can accept `title` as either string or a node, to make it backwards compatible.
> 
> @mihirsoni and @ylwu-amzn feel free to let me know if that sounds good to you

Sounds good to me as well.